src/main/groovy/org/mockito/release/internal/gradle/BintrayPlugin.java

```diff
@@ -41,20 +41,26 @@ public void apply(final Project project) {
         // this way we avoid version conflicts and any bintray dependencies for users who don't use bintray
         project.getPlugins().apply("com.jfrog.bintray");
 
+        //Configure some properties right away
+        final BintrayExtension bintray = project.getExtensions().getByType(BintrayExtension.class);
+        bintray.setPublish(true);
+
         final BintrayUploadTask bintrayUpload = (BintrayUploadTask) project.getTasks().getByName(BINTRAY_UPLOAD_TASK);
 
         bintrayUpload.doFirst(new Action<Task>() {
             public void execute(Task task) {
                 //TODO unit test
                 BintrayUploadTask t = (BintrayUploadTask) task;
+
+                //workaround for https://github.com/bintray/gradle-bintray-plugin/issues/170
```

Comment:
Given that your primary use case is making ```./gradlew tasks``` work, I suggest we use "lazyConfiguration" in this case. You can refer to the javadoc for LazyConfiguration class. If it is fuzzy, let me know so we make it clear! I love good docs ;)